lib/core/include/irods/process_stash.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef IRODS_PROCESS_STASH_HPP
+#define IRODS_PROCESS_STASH_HPP
+
+/// \file
+
+// Boost.Any is used instead of the implementation provided by the standard library
+// because it produces the correct results when used across shared library boundaries.
+#include <boost/any.hpp>
+
+#include <string>
+
+/// \since 4.2.12
+namespace irods::process_stash
```

Comment:
Add the following function:
- `auto handles() -> std::vector<std::string>`: Returns all handles in the stash.

With this function, developers can detect the size of the stash and fetch all objects within the stash.

Reply:
Done.